style.css

```diff
@@ -0,0 +1,52 @@
+@import 'https://fonts.googleapis.com/css?family=Lato:400,700';
+
+#nav-bar {
+  display: table-column;
+  position: absolute;
+  margin-left: 40%;
+  margin-top: 15%;
+}
+
+body {
+  background-color: #eee;
+  font-family: 'Lato', sans-serif;
+}
+
+a {
+  color: #000;
+  text-decoration: none;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  min-height: 75px;
+  padding: 20px;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  background-color: #eee;
+  text-decoration: none;
+}
+
+header img {
+  width: 400px;
+  margin-left: 0;
+}
+
+header li {
+  list-style: none;
+  display: inline-block;
+  list-style-type: none;
+}
+
+.hero {
+  display: flex;
+  justify-content: space-around;
+}
+
+@media (min-width: 500px) {
+  body {
+    width: 70%;
+  }
```

Comment:
The Min-width of 500px is quite limiting as trend now priorotizes mob devices. I would suggest 320px in that regard